jest.config.js

```diff
@@ -2,209 +2,190 @@
 
-  // An array of file extensions your modules use
-  moduleFileExtensions: [
-    "js",
-    "json",
-    "jsx",
-    "ts",
-    "tsx",
-    "node"
-  ],
+	// An object that configures minimum threshold enforcement for coverage results
+	// coverageThreshold: undefined,
 
-  // A map from regular expressions to module names that allow to stub out resources with a single module
-  moduleNameMapper: {
-    '\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
-      '<rootDir>/__mocks__/fileMock.js',
-    '\\.(css|scss|sass)$': 'identity-obj-proxy'},
-  
+	// A path to a custom dependency extractor
+	// dependencyExtractor: undefined,
 
-  // An array of regexp pattern strings, matched against all module paths before considered 'visible' to the module loader
-  // modulePathIgnorePatterns: [],
+	// Make calling deprecated APIs throw helpful error messages
+	// errorOnDeprecated: false,
 
-  // Activates notifications for test results
-  // notify: false,
+	// Force coverage collection from ignored files using an array of glob patterns
+	// forceCoverageMatch: [],
 
-  // An enum that specifies notification mode. Requires { notify: true }
-  // notifyMode: "failure-change",
+	// A path to a module which exports an async function that is triggered once before all test suites
+	// globalSetup: undefined,
 
-  // A preset that is used as a base for Jest's configuration
-  // preset: undefined,
+	// A path to a module which exports an async function that is triggered once after all test suites
+	// globalTeardown: undefined,
 
-  // Run tests from one or more projects
-  // projects: undefined,
+	// A set of global variables that need to be available in all test environments
+	// globals: {},
 
-  // Use this configuration option to add custom reporters to Jest
-  // reporters: undefined,
+	// The maximum amount of workers used to run your tests. Can be specified as % or a number. E.g. maxWorkers: 10% will use 10% of your CPU amount + 1 as the maximum worker number. maxWorkers: 2 will use a maximum of 2 workers.
+	maxWorkers: '50%',
 
-  // Automatically reset mock state between every test
-  // resetMocks: false,
+	// An array of directory names to be searched recursively up from the requiring module's location
+	moduleDirectories: ['node_modules'],
 
-  // Reset the module registry before running each individual test
-  // resetModules: false,
+	// An array of file extensions your modules use
+	moduleFileExtensions: ['js', 'json', 'jsx', 'ts', 'tsx', 'node'],
 
-  // A path to a custom resolver
-  // resolver: undefined,
+	// A map from regular expressions to module names that allow to stub out resources with a single module
+	moduleNameMapper: {
+		'\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$':
+			'<rootDir>/__mocks__/fileMock.js',
+		'\\.(css|scss|sass)$': 'identity-obj-proxy',
+	},
+	coveragePathIgnorePatterns: [
+		'/node_modules/',
```

Comment:
node_modules and test has already been ignored, why adding it here again twice, also you shouldn't ignore the store, please take it out

Reply:
Hey, @Ruqoyah  I have completed all the feedback, now you can check.